src/figuresClasses.ts

```diff
@@ -1,11 +1,112 @@
-export interface Figure {}
+export interface Figure {
+  color: string;
+  a?: number;
+  b?: number;
+  c?: number;
+  radius?: number;
+  width?: number;
+  height?: number;
```

Comment:
all these fields are redundant and don't forget to add shape here

Reply:
Why are they redundant? We use them all in classes